<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1808</width>
    <height>959</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Роботизированный комплекс лазерной обработки</string>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CMainViewport" name="mainView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1808</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Вид</string>
    </property>
    <widget class="QMenu" name="menuMSAA">
     <property name="title">
      <string>MSAA</string>
     </property>
     <addaction name="actionMSAA_Off"/>
     <addaction name="actionMSAA_2X"/>
     <addaction name="actionMSAA_4X"/>
     <addaction name="actionMSAA_8X"/>
    </widget>
    <addaction name="actionShading"/>
    <addaction name="menuMSAA"/>
    <addaction name="actionFPS"/>
    <addaction name="separator"/>
    <addaction name="actionClearJrnl"/>
    <addaction name="actionCalib"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справ&amp;ка</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockSettings">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Калибровка</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="CGuiSettingsWidget" name="wSettings"/>
  </widget>
  <widget class="QDockWidget" name="dockJournal">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="teJrnl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Source Code Pro'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Импорт...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCalibApend">
   <property name="text">
    <string>Добавить точку</string>
   </property>
  </action>
  <action name="actionCalibReset">
   <property name="text">
    <string>Сброс</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О Программе</string>
   </property>
  </action>
  <action name="actionShading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Шейдинг</string>
   </property>
  </action>
  <action name="actionCalib">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Калибровка</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="actionMSAA_Off">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Откл</string>
   </property>
  </action>
  <action name="actionMSAA_2X">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MSAA 2X</string>
   </property>
  </action>
  <action name="actionMSAA_4X">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MSAA 4X</string>
   </property>
  </action>
  <action name="actionMSAA_8X">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MSAA 8X</string>
   </property>
  </action>
  <action name="actionFPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Счетчик FPS</string>
   </property>
  </action>
  <action name="actionClearJrnl">
   <property name="text">
    <string>Очистить журнал</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMainViewport</class>
   <extends>QWidget</extends>
   <header>cmainviewport.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CGuiSettingsWidget</class>
   <extends>QWidget</extends>
   <header>cguisettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
